### ISSUE TITLE
AMO: PayPal "Contribute" iframe silently fails

### ISSUE THREAD

**Issue by ronjouch** [mozilla/addons#240]
### Describe the problem and steps to reproduce it:
Try to donate to any addon. For example, go to the [VimFx page](https://addons.mozilla.org/en-US/firefox/addon/vimfx/), click `Contribute`, keep the default amount, and click `Make Contribution`.

### What happened?

* No PayPal iframe loads
* The full-page gray overlay (loaded after clicking `Make Contribution`) stays active, preventing clicking on anything (the only "exit" from this state is a refresh)
* In the DevTools console,

```
09:38:24.227 Load denied by X-Frame-Options: https://www.paypal.com/us/cgi-bin/webscr?cmd=_dispatch-failed does not permit cross-origin framing. 1 (unknown)
```

### What did you expect to happen?

Seeing the PayPal iframe.

### Anything else we should know?

* Firefox Developer Edition 51.0a2 (2016-10-01) (64-bit) under Arch Linux, with a bunch of config and addons.
* Deactivating all my addons didn't help.
* Testing under a fresh Firefox or Chrome profile works.
* Feel free to ask for about:config chunks or any debug info.
* Iframes from other websites (e.g. youtube, bandcamp) *do* work.

I'm filing this bug hoping that:

1. Someone can help me narrow down on the cause of this problem (Firefox config, probably?).
2. You can either fix the problem, or catch the error and display an error message suggesting potential causes.

**Comment by muffinresearch** [comment]
@ronjouch This can often be a temporary failure with Paypal's service. Can you try again and let us know if it's still failing?

**Comment by ronjouch** [comment]
> @ronjouch This can often be a temporary failure with Paypal's service. Can you try again and let us know if it's still failing?

Hi. Yes it is still failing under my configured Firefox profile, and still working in a fresh profile. It's very reproducible on my setup, it's clearly not a failure on PayPal's side.

**Comment by andymckay** [comment]
https://bugzilla.mozilla.org/show_bug.cgi?id=896653 "clearly not a failure on PayPal's side" is debatable, but I don't know where the problem is.

**Comment by ronjouch** [comment]
> "clearly not a failure on PayPal's side" is debatable

@andymckay indeed; I was contradicting a hypothesis from @muffinresearch but did a bad job at it. Rephrasing, using his terms: this is clearly not *"a temporary failure with Paypal's service"*, as it's 100% reproducible on my {configuredFx vs. freshFx|Chrome} setup.

Thanks for the BugZilla link, yup it looks like a duplicate. So far I only did minimum research (disabling addons and testing with a fresh profile), hoping to get suggestions. Closing for now, I'll try to get a minimal use case (bringing my configuredFx options to freshFx and binary-searching for a culprit, other suggestions welcome) and will reopen if I'm able to find one.

Thanks!
